README.rst

```diff
@@ -20,6 +24,16 @@ Usage
    12345
    ['Marxus, Karlus', 'Lindgren, Astrid']
 
+Writing:
+
+.. code:: python
+
+   >>> import os
+   >>> import rispy
+   >>> filepath = 'tests/example_export_full.ris'
+   >>> with open(filepath, 'w') as bibliography_file:
+   ...     rispy.dump(entries, bibliography_file)
```

Comment:
could you create a simple `entries` object in the docs to demonstrate the minimal requirements?